test/NonSilo.Tests/Serialization/BuiltInSerializerTests.cs

```diff
@@ -129,6 +130,9 @@ public void InternalSerializableTypesHaveSerializers()
             Assert.True(
                 environment.SerializationManager.HasSerializer(grainReferenceType),
                 $"Should be able to serialize grain reference type {grainReferenceType}.");
+            Assert.True(
+                environment.SerializationManager.HasSerializer(typeof(PubSubGrainState)),
+                $"Should be able to serialize internal type {nameof(PubSubGrainState)}.");
```

Comment:
StreamSubscriptionHandle is another class which is Serializable in streaming, and people may persisted it .. it's valuable to add that too. 
I'm not familiar with other valuable serializable which worth to add this test. But I just did a search on [Serializable] in our repo, there's a lot. Maybe we should add more coverage on serialization on internal types